### PR TITLE
Bw 442

### PR DESCRIPTION
**Pull request by milansar**

### CODE REVIEW

File: functions/controllers/user.controller.ts [hunk 1]
```diff
@@ -99,6 +99,8 @@ function setUser(user, res) {
             UserControllerConstants.mailSubject, htmlContent);
         mail.sendMail();
     }
+
+    delete user['roles'];
```
Comment (anihalaney) [inline functions/controllers/user.controller.ts]: what is this for?

**Comment by anihalaney** [comment]
how about firestore rules?

File: functions/controllers/user.controller.ts (hunk 1 above)
Comment (milansar) [reply functions/controllers/user.controller.ts]: This is to ensure client App are not able to overwrite roles information from user document. firestore calls from API is not governed by firestore rules

File: firestore-rules.txt [hunk 2]
```diff
@@ -119,6 +125,12 @@ service cloud.firestore {
       }      
     }
     
+    match /accounts {    
+      match /{document=**} {
+        allow read,get:  if true;
```
[the remark this answers is not in the document]
Comment (milansar) [reply firestore-rules.txt]: update to rule so only user itself can access account document

**Pull request merged**